docs/source/maintainers.rst

```diff
@@ -0,0 +1,63 @@
+=====================
+Maintainer Guidelines
+=====================
+
+This page describes best practices for Dask maintainers.
+
+Thank you for helping make Dask a useful library and fostering a
+welcoming community.
+
+Merging pull requests
+=====================
+
+Pull requests should be reviewed
+--------------------------------
+
+Pull requests from non-maintainers should be reviewed and approved by at
+least one maintainer before being merged.
+
+Ideally, pull requests from maintainers should also be reviewed before
+merging. However, because reviewer bandwidth is limited, maintainers will
+sometimes self-merge their own pull requests if the content of the pull
+request is viewed as uncontroversial (e.g. typo fix). If a maintainer has
+a substantial pull request which hasn't received a review, and the maintainer
```

Comment:
I think most of these situations occur because people simply do not know that there is a small PR that can be easily reviewed and merged.
How about instead tagging a team to get feedback faster?

Reply:
Based on @jsignell's feedback here https://github.com/dask/dask/pull/9309#discussion_r930087194 I updated this section to also include a "ping a team or individual maintainer" step.

I agree the ideal case is that all PRs are reviewed before being merged. This is also what happens for almost all PRs. That said, sometimes people are busy with other things and reviews are hard to come by. Similar to https://github.com/dask/dask/pull/9309#discussion_r951889892, I'd prefer to move forward with this section as is (since it accurately describes our current behavior) and, if desired, opening up a separate conversation around changing this practice